views/includes/start-appoint-remove.html

```diff
@@ -8,8 +8,8 @@ <h1 class="govuk-heading-xl">
     <li>{{ i18n.startUseThisServiceInfo4 }}</li>
 </ul>
 <p class="govuk-body">{{ i18n.startUseThisServiceInfo5 }}
-    <a href="https://ewf.companieshouse.gov.uk//seclogin?tc=1" data-event-id="web-filing-link-clicked" class="govuk-link">{{ i18n.startUseThisServiceInfo6 }}</a> {{ i18n.startUseThisServiceInfo7 }}
-    <a href="https://www.gov.uk/government/collections/companies-house-forms-for-limited-companies#directors-and-secretaries" data-event-id="companies-house-form-link-clicked" class="govuk-link">{{ i18n.startUseThisServiceInfo8 }}</a>.
+    <a href="https://ewf.companieshouse.gov.uk//seclogin?tc=1/#lang-{{lang|default('en')}}" data-event-id="web-filing-link-clicked" class="govuk-link">{{ i18n.startUseThisServiceInfo6 }}</a> {{ i18n.startUseThisServiceInfo7 }}
```

Comment:
I tried this manually and it's not translated for me.

Reply:
@AndreKramer7654 Hnmmm that is true it is not translated on a freshly loaded page same with your suggested link. It only translates if there is a cache on that page (if you try in a private window or empty cache, it does not translate)

Reply:
Tried this again and the welsh redirects to:
https://idam-ui.company-information.service.gov.uk/#lang-cy
and english to:
https://idam-ui.company-information.service.gov.uk/#lang-en
But we should ask CH if these are the correct urls as the hostname is different. Not sure how ?lang=cy was working or why it did not redirect me to cy yesterday for:
https://ewf.companieshouse.gov.uk//seclogin?tc=1/#lang-cy

Reply:
Identified as not in our control